mmv1/products/clouddeploy/api.yaml

```diff
@@ -0,0 +1,234 @@
+              Stage specifies a location to which to deploy.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: 'targetId'
+                  required: true
+                  input: true
+                  description: |
+                    The targetId to which this stage points. This field refers exclusively to the last segment of a target name.
+                    For example, this field would just be my-target (rather than projects/project/locations/location/targets/my-target).
+                    The location of the Target is inferred to be the same as the location of the DeliveryPipeline that contains this Stage.
+                - !ruby/object:Api::Type::Array
+                  name: 'profiles'
+                  item_type: Api::Type::String
+                  input: true
+                  description: |
+                    Skaffold profiles to use when rendering the manifest for this stage's Target.
+  - !ruby/object:Api::Resource
+    name: 'Target'
+    base_url: projects/{{project}}/locations/{{region}}/targets
+    self_link: 'projects/{{project}}/locations/{{region}}/targets/{{name}}'
+    create_url: projects/{{project}}/locations/{{region}}/targets?targetId={{name}}
+    input: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/deploy/docs'
+      api: 'https://cloud.google.com/deploy/docs/api/reference/rest/v1/projects.locations.targets'
+    description: |-
+      A Target defines a location to which a Skaffold configuration can be deployed.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: region
+        description: The region of the Target Store. eg us-central1
+        url_param_only: true
+        input: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: The name of the Target. This value may be up to 60 characters, and valid characters are [a-z0-9_]. The first character cannot be a number.
+        input: true
+        url_param_only: true
+        pattern: projects/{{project}}/locations/{{region}}/targets/{{name}}
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: Description of the Target.
+        input: true
+      - !ruby/object:Api::Type::String
+        name: 'createTime'
+        output: true
+        description: |
+          The timestamp of when the Target was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      - !ruby/object:Api::Type::String
+        name: 'updateTime'
+        output: true
+        description: |
+          The timestamp of when the Target was last updated in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      - !ruby/object:Api::Type::String
+        name: 'etag'
```

Comment:
Is this needed for any particular reason? We tend to avoid adding `etag` and some of these other output-only fields if they aren't needed for other resources that depend on this. Similarly, updateTime & createTime can be removed

Reply:
Were these needed?